apps/betterangels-backend/notes/tests/test_queries.py

```diff
@@ -340,6 +376,68 @@ def test_notes_query_filter_by_teams(
             for idx, note_label in enumerate(returned_note_labels):
                 self.assertEqual(notes[idx]["id"], getattr(self, note_label)["id"])
 
+    @parametrize(
+        ("name_search, expected_results_count, expected_authors"),
+        [
+            ("Maximoff", 1, ["interaction_author_2"]),
+            ("Pietro Maximoff", 0, None),
+            (
+                "Alex",
+                2,
+                ["interaction_author", "interaction_author_3"],
```

Comment:
if you remove this comma, it'll collapse this tuple onto one line
```suggestion
                ["interaction_author", "interaction_author_3"]
```